src/pages/AppDetail.svelte

```diff
@@ -58,7 +59,12 @@
   <TabGroup
     active={$pageState.itemType}
     on:tabChanged={({ detail }) => {
-      pageState.set({ ...$pageState, itemType: detail.active, search: "" });
+      pageState.set({
+        ...$pageState,
+        itemType: detail.active,
+        search: "",
+        page: "",
+      });
```

Comment:
Small nit: let's try to keep state resetting similar to the initial default state that we set above.

```suggestion
      pageState.set({
        ...$pageState,
        itemType: detail.active,
        search: "",
        page: 1,
      });
```